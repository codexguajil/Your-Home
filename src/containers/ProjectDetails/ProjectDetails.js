import React from 'react';
import RoomsContainer from '../../components/RoomsContainer/RoomsContainer'

class ProjectDetails extends React.Component {

  render() {
    const { name, address, city, description, end_date, rooms, start_date, state, zip_code } = this.props.project
    
    return (
      <div>
        <div className='project-details'>
          <h2>{name}</h2>
          {/* <p>{address}, {city}, {state}, {zip_code}</p> */}
          <p>{address}</p>
          <p>{description}</p>
          {/* <p><span>{start_date}</span> to <span>{end_date}</span></p> */}
        </div>
        <RoomsContainer rooms={rooms}/>
      </div>
    )
  }
}

export default ProjectDetails;