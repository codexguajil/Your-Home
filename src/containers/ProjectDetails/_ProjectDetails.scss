.project-page {
  margin-bottom: 40px;
  display: flex;
  justify-content: space-around;
  align-items: center;

  .basic {
    display: flex;
    flex-direction: column;
    align-items: center;
  }
  
  h2 {
    font-size: 200%;
  }

  .header-btn {
    width: 240px;
  }

  p {
    margin-bottom: 8px;
  }

}