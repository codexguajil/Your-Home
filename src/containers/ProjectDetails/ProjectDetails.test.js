import React from 'react';
import { shallow } from 'enzyme';
import ProjectDetails from './ProjectDetails';

describe('ProjectDetails', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<ProjectDetails/>)
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

});