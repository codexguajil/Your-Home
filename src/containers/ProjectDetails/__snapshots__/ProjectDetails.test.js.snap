// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectDetails should match snapshot 1`] = `
<div>
  <div>
    <div>
      <div
        className="project-page"
      >
        <ConnectFunction
          className="add-room-btn"
          id={1}
        />
        <div
          className="basic"
        >
          <h2>
            House 1
          </h2>
          <p>
            123 Fake St.
          </p>
          <p>
            Big, white house
          </p>
        </div>
        <ConnectFunction
          address="123 Fake St."
          className="edit-project-btn"
          description="Big, white house"
          id={1}
          name="House 1"
        />
      </div>
    </div>
  </div>
</div>
`;
